resources/views/home.blade.php

```diff
@@ -1,50 +1,182 @@
+                        <feGaussianBlur stdDeviation="3"/>
+                        <feColorMatrix values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.05 0"/>
+                        <feBlend in2="BackgroundImageFix" result="effect1_dropShadow"/>
+                        <feColorMatrix in="SourceAlpha" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/>
+                        <feOffset dy="10"/>
+                        <feGaussianBlur stdDeviation="7.5"/>
+                        <feColorMatrix values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.1 0"/>
+                        <feBlend in2="effect1_dropShadow" result="effect2_dropShadow"/>
+                        <feBlend in="SourceGraphic" in2="effect2_dropShadow" result="shape"/>
+                    </filter>
+                    </defs>
+                </svg>
+            </div>
         </div>
     </div>
-</div>
+</section>
+
 
-<div class="container-fluid bg-white">
-    <div class="row justify-content-center">
-        <div class="col-md-12 my-5">
-            <h1 class="text-center">
-                @lang('home.text_subscribe_cta', ['link' => 'sms://+1' . env('TWILIO_FROM_NUMBER') . ';?&body=' . __('home.text_subscribe_keyword')])
-            </h1>
-            <h2 class="text-center">
-                @lang('home.add_to_contacts_cta', ['link' => 'tel:+1' . env('TWILIO_FROM_NUMBER')])
-            </h2>
+<section id="steps" class="max-w-5xl mx-auto px-8 py-10 sm:py-20">
+    <div class="sm:flex -mx-4">
+        <div class="sm:w-1/3 px-4 mb-8 sm:mb-0">
+            <h2 class="text-2xl font-medium font-display">Step 1</h2>
+            <p>@lang('home.step_1')</p>
         </div>
-    </div>
-    <div class="row justify-content-center">
-        <div class="col-md-12 my-5">
-            <h1 class="text-center">
-                <strong>@lang('home.locale_support_head')</strong>
-            </h1>
-            <h4 class="text-center">
-                @lang('home.locale_support_body', ['link' => 'tel:+1' . env('TWILIO_FROM_NUMBER')])
-            </h4>
+        <div class="sm:w-1/3 px-4 mb-8 sm:mb-0">
+            <h2 class="text-2xl font-medium font-display">Step 2</h2>
+            <p>@lang('home.step_2')</p>
+        </div>
+        <div class="sm:w-1/3 px-4">
+            <h2 class="text-2xl font-medium font-display">Step 3</h2>
+            <p>@lang('home.step_3')</p>
         </div>
     </div>
-</div>
+</section>
 
-<div class="container-fluid">
-    <div class="row justify-content-center">
-        <div class="col-md-12 my-5">
-            <h2 class="text-center">@lang('home.partners_head')</h2>
-            <p class="text-center">
-                @lang('home.partners_body')
-            </h2>
-        </div>
+<section class="px-8 py-16 sm:py-32 bg-red-200 relative">
+    <div class="absolute inset-0" style="background-image: url({{ asset('img/map.jpg') }})"></div>
+    <div class="bg-white max-w-md mx-auto px-10 py-5 text-center rounded-lg shadow-lg relative z-10">
```

Comment:
Tailwinds seems interesting! Lots of small single-purpose classes it looks like?